scipy/stats/_distribution_infrastructure.py

```diff
@@ -3497,17 +3517,21 @@ def make_distribution(dist):
 
             endpoints : tuple
                 A tuple defining the lower and upper endpoints of the domain of the
-                parameter. Allowable values are floats or the name (string) of another
+                parameter; allowable values are floats or the name (string) of another
                 parameter.
 
             inclusive : tuple of bool
                 A tuple specifying whether the endpoints are included within the domain
                 of the parameter.
 
-        support : tuple
+        support : tuple or method.
             A tuple defining the lower and upper endpoints of the support of the
-            distribution. Allowable values are floats or the name (string) of a
-            parameter.
+            distribution or a callable which calculates such a tuple based on the
+            parameter values. If a tuple, values can be floats or the name (string)
+            of a parameter. If ``support`` is defined as a method, it must take the
+            parameter names as keyword arguments and return a 2-tuple giving the
+            numerical lower and upper endpoints of the support for fixed values of
+            the parameters.
```

Comment:
I need to document that if a `method` is passed for  `support`, it must accept array arguments for the parameters, and in that case return a tuple of arrays.